docs/source/models/supported_models.rst

```diff
@@ -309,6 +309,11 @@ Text Generation
     - :code:`upstage/solar-pro-preview-instruct`, etc.
     - ✅︎
     - ✅︎
+  * - :code:`TeleChat2ForCausalLM`
+    - TeleChat2
+    - :code:`TeleAI/TeleChat2-3B`, :code:`TeleAI/TeleChat2-7B`, :code:`TeleAI/TeleChat2-35B`, etc.
+    - 
```

Comment:
Oh, we need to handle `lm_head` with PP and LoRA config for `TeleChat2ForCausalLM` in intialization, because we overwrite the `__init__` from `LlamaForCausalLM`.

Let me make a quick fix for it.

Reply:
`TeleChat2ForCausalLM` should also support PP now.